marlowe/cli-doc/simple-test.sh

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env bash
+
+
+# Simple end-to-end test of `marlowe-cli`.
+#
+#
+# The environment variable CARDANO_NODE_SOCKET_PATH must be set.
+#
+# The following tools must be on the path:
+#   marlowe-cli
+#   cardano-cli
+#   sed
+#   jq
+#   timeout
+#
+# This script exits with an error value if the end-to-end test fails.
+
+
+set -ex
+
+
+# Select the network.
+
+NETWORK=testnet
+MAGIC_FLAG=--testnet-magic
+MAGIC_NUM=1097911063
```

Comment:
It's awkward to have both `MAGIC_FLAG` and `MAGIC_NUM`. Change this back to just `MAGIC`, and use `"${MAGIC[@]}"` instead of `"$MAGIC_FLAG" "$MAGIC_NUM"` in all of the scripts.

Reply:
Fixed in a8d9ad37ba44ff217924b1b0edab1b305707ffc3.